apps/web/components/getting-started/steps-views/SetupAvailability.tsx

```diff
@@ -21,7 +21,7 @@ const SetupAvailability = (props: ISetupAvailabilityProps) => {
   const { nextStep } = props;
 
   const queryAvailability = trpc.viewer.availability.schedule.get.useQuery(
-    { scheduleId: defaultScheduleId! },
+    { scheduleId: defaultScheduleId ?? undefined },
```

Comment:
@sean-brydon Is this what we want? Or should this throw an error if defaultScheduleId is null/undefined?

Reply:
Talked to Sean. All good.